test/penalizer/Penalizer.test.ts

```diff
@@ -389,7 +379,6 @@ async function assertTransactionFails (rawTx: string, reason?: string): Promise<
   } catch (err) {
     if (reason === undefined || reason === null) {
       // we don't want to check why the transaction failed, but only if failed or not.
-      assert(true)
```

Comment:
please let me know if we need this one (and if possible, why)

Reply:
I don't like it at all, but I introduced this change to use the same function from [here](https://github.com/mortelli/rif-relay/blob/feature/PP-52/penalizer-time-verification/test/penalizer/Penalizer.test.ts#L354).
In that case, we don't have a specific message to check for, so we can only check that it fails. 
Any suggestions are more than welcome!